ui/admin/app/components/form/role/grants/index.hbs

```diff
@@ -13,25 +13,26 @@
     <Rose::List::KeyValue as |list|>
       {{! Add grant field (only one per form) }}
       <list.item as |item|>
-        <form.input
-          @contextual={{true}}
-          @name='grant'
-          @type='text'
-          @value={{this.newGrantString}}
-          as |input|
-        >
+        <form.input @contextual={{true}}>
           <item.key>
-            <input.label>{{t
-                'resources.role.grant.actions.create'
-              }}</input.label>
+            <Hds::Form::Label @controlId='add-grant'>
+              {{t 'resources.role.grant.actions.create'}}
+            </Hds::Form::Label>
           </item.key>
           <item.cell>
-            <input.field title={{t 'form.grant.help'}} />
+            <Hds::Form::TextInput::Field
+              {{on 'input' (set-from-event this 'newGrantString')}}
+              name='grant'
+              @id='add-grant'
+              @type='text'
+              @value={{this.newGrantString}}
+              @extraAriaDescribedBy={{t 'form.grant.help'}}
```

Comment:
What are these `@extraAriaDescribedBy` for? These attach extra IDs for [aria-describedBy](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby) which take in IDs, not internationalized text.

Reply:
In retrospect, it looks like we had a title because there was no where to add a description for the input, so I think we should keep the title. It'll just be a description when you hover the input which we normally don't do but it's better than nothing for these input fields

Reply:
That's fair